snuba/query/snql/parser.py

```diff
@@ -617,11 +618,25 @@ def visit_totals_clause(
     def visit_limit_by_clause(
         self,
         node: Node,
-        visited_children: Tuple[Any, Any, Any, Literal, Any, Any, Any, Column],
+        visited_children: Tuple[
+            Any, Any, Any, Literal, Any, Any, Any, Column, Sequence[Column]
```

Comment:
This typing isn't quite right.
```suggestion
            Any, Any, Any, Literal, Any, Any, Any, Column, Optional[Sequence[Column]]
```